tests/sentry/api/endpoints/test_project_stacktrace_link.py

```diff
@@ -116,10 +116,8 @@ def test_file_not_found_error(self):
         )
         assert response.data["config"] == self.expected_configurations(self.code_mapping1)
         assert not response.data["sourceUrl"]
-        # XXX: This depends on what was the last attempted code mapping
-        assert response.data["error"] == "stack_root_mismatch"
+        assert response.data["error"] == "file_not_found"
```

Comment:
In #41409 I added more than one code mapping and it showed that the error value would depend on the last attempted code mapping rather than the last one that _matched_.

This PR fixes the issue since the first code mapping matches the stack trace root and the source file is not found (as the original test was):
https://github.com/getsentry/sentry/blob/78a95d5448ca23e13f5cc1cca0ad37bbca140763/tests/sentry/api/endpoints/test_project_stacktrace_link.py#L58